tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Also, would be better fot you to add eslint for the project - https://javascript.plainenglish.io/setting-eslint-and-prettier-on-a-react-typescript-project-2021-22993565edf9